infra/scripts/bash-helpers.sh

```diff
@@ -97,53 +135,146 @@ addr-from-dockerhost () {
 }
 
 vic-tail-portlayer() {
+    vicProfileTranscode
+
     unset keyarg
     if [ -e "$HOME"/.ssh/authorized_keys ]; then
         keyarg="--authorized-key=$HOME/.ssh/authorized_keys"
     fi
 
-    out=$("$(vic-path)"/bin/vic-machine-"$OS" debug --target="$GOVC_URL" --compute-resource="$COMPUTE" --name="${VIC_NAME:-${USER}test}" --enable-ssh "$keyarg" --rootpw=password --thumbprint="$THUMBPRINT" "$@")
+    out=$("$(vic-path)"/bin/vic-machine-"$OS" debug --target="$TARGET_URL" --compute-resource="$COMPUTE" --name="${VIC_NAME:-${USER}test}" --enable-ssh "$keyarg" --rootpw=password --thumbprint="$THUMBPRINT" "$@")
     host=$(echo "$out" | grep DOCKER_HOST | awk -F"DOCKER_HOST=" '{print $2}' | cut -d ":" -f1 | cut -d "=" -f2)
 
     sshpass -ppassword ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@"${host}" tail -f /var/log/vic/port-layer.log
 }
 
 vic-tail-docker() {
+    vicProfileTranscode    
+
     unset keyarg
     if [ -e "$HOME"/.ssh/authorized_keys ]; then
         keyarg="--authorized-key=$HOME/.ssh/authorized_keys"
     fi
 
-    out=$("$(vic-path)"/bin/vic-machine-"$OS" debug --target="$GOVC_URL" --compute-resource="$COMPUTE" --name="${VIC_NAME:-${USER}test}" --enable-ssh "$keyarg" --rootpw=password --thumbprint="$THUMBPRINT" "$@")
+    out=$("$(vic-path)"/bin/vic-machine-"$OS" debug --target="$TARGET_URL" --compute-resource="$COMPUTE" --name="${VIC_NAME:-${USER}test}" --enable-ssh "$keyarg" --rootpw=password --thumbprint="$THUMBPRINT" "$@")
     host=$(echo "$out" | grep DOCKER_HOST | awk -F"DOCKER_HOST=" '{print $2}' | cut -d ":" -f1 | cut -d "=" -f2)
 
     sshpass -ppassword ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@"${host}" tail -f /var/log/vic/docker-personality.log
 }
 
+# turns the configuration variables into the ones expected by the vic-x functions
+vicProfileTranscode() {
```

Comment:
Could you rename it to: "initVicProfileVariables" or something better that could be more clear what it does by its name?
It was not initially clear to me what "vicProfileTranscode" does, so I was looking for this function specifically to understand what is going on.

Reply:
I'll update the comment to explain more - it's transcoding the variables from the profile definition to the ones the helper expects so that existing profiles in the old form will still work.